src/traffic_server/traffic_server.cc

```diff
@@ -2125,6 +2136,9 @@ main(int /* argc ATS_UNUSED */, const char **argv)
 
     ink_set_thread_name("[TS_MAIN]");
 
+    if (ports_are_open && cache_is_initialized) {
+      Note("traffic server initialized");
```

Comment:
What about Au tests that disable caching?  https://github.com/apache/trafficserver/blob/c7e80542aa1a5323399226f636ef196955f60791/tests/gold_tests/continuations/openclose.test.py#L46 to give one example.  Will this message still be output when cache is disabled?

It looks like this Au tests (openclose) explicitly uses port probing for the ready test for TS.  Is the plan to use a non-default test for test cases that disable caching?

Reply:
I was concerned about this too, but it turns out that the cache initialized callback is called even if http.cache.http is set to 0 (http caching is disabled), so this message can be used in those cases too.